### DISCUSSION TITLE
Maximize/Restore Should Be The Same Shortcut

### DISCUSSION THREAD

**Discussion by dthom264** [rxhanson/Rectangle#418] [Ideas]
Currently Maximize and Restore use two different shortcuts. However, it would be more convenient if Rectangle recognized which state the window is currently in (maximized or not maximized), and when the user presses the shortcut, the state is changed to whichever state the window is not currently in.

**Comment by rxhanson** [comment]
Makes sense to me. If anyone's interested in contributing, this would be a good first pull request to make since it's not too involved.
